app/views/content/is-teaching-right-for-me/languages/_article.html.erb

```diff
@@ -0,0 +1,105 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>Being a modern languages teacher is rewarding as you can open young minds to the languages and cultures around the world.</p>
+  <p>There are 18 modern languages you could teach with the most popular being French, German and Spanish.</p>
+  <p>Tax-free bursaries of £25,000 are available for eligible trainee Spanish, German and French teachers. Scholarships of £27,000 are available for eligible trainee modern languages teachers.</p>
```

Comment:
@caitlin-reidy This is now the wrong way round - bursaries are for all languages teachers, but scholarships are only for French, German and Spanish (as per later down on the page)